linux/src/threadpool_linux.c

```diff
@@ -77,6 +94,19 @@ typedef struct THREADPOOL_TAG
     THREAD_HANDLE* thread_handle_array;
 } THREADPOOL;
 
+void on_timer_callback(sigval_t timer_data)
```

Comment:
>on_timer_callback [](http://example.com/codeflow?start=5&length=17)

how come this function doesn't decrement pending_api_call??

Reply:
All removed.